app/lib/features/home/pages/dashboard.dart

```diff
@@ -138,14 +138,17 @@ class _DashboardState extends ConsumerState<Dashboard> {
                         mode: DisplayMode.User,
                       ),
                     ),
-                    child: Container(
-                      key: Keys.avatar,
-                      margin: const EdgeInsets.all(8),
-                      child: InkWell(
-                        onTap: () => context.pushNamed(Routes.myProfile.name),
-                        child: const UserAvatarWidget(size: 20),
-                      ),
-                    ),
+                    child: !isDesktop
```

Comment:
arguably this should depend on the width/whether the sidebar is available rather than it "being desktop", but sure, let's do this for now...